.github/workflows/continuous-build.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Continuously build images
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 0 * * 0"
+
+jobs:
+  run_image_update_builds:
+    strategy:
+      matrix:
+        version:
+          - major: 8
+            full: "8.6"
+          - major: 9
+            full: "9.0"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to Quay
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAYIO_USERNAME }}
+          password: ${{ secrets.QUAYIO_TOKEN }}
+
+      - name: Build and push not latest version
+        if: matrix.version.major != 9
+        uses: docker/build-push-action@v3
+        with:
+          file: Dockerfile-update
+          platforms: linux/amd64,linux/arm64/v8
+          push: true
+          tags: |
+            rockylinux/rockylinux:${{ matrix.version.major }}
+            quay.io/rockylinux/rockylinux:${{ matrix.version.major }}
+          build-args: |
+            Imageversion=${{ matrix.version.full }}
+
+      - name: Build and push latest version
+        if: matrix.version.major == 9
+        uses: docker/build-push-action@v3
+        with:
+          file: Dockerfile-update
+          platforms: linux/amd64,linux/arm64/v8
+          push: true
+          tags: |
+            rockylinux/rockylinux:${{ matrix.version.major }}
+            rockylinux/rockylinux:latest
```

Comment:
We aren't currently doing latest tags as it's ambiguous as to what it refers to when we have multiple major versions tagged. E.g., it's "supposed" to be just the most-recently-pushed tag in the 'traditional' sense.

Additionally, I'd like to support a MAJ.MIN.IS8601DATE tag, moving only the MAJ tag to the latest version each time.